tensorflow/python/ops/image_ops_impl.py

```diff
@@ -947,6 +947,119 @@ def resize_images(images,
     return images
 
 
+@tf_export('image.resize_image_aspect_with_pad')
+def resize_image_aspect_with_pad(image, target_height, target_width,
+                                 method=ResizeMethod.BILINEAR):
+  """
+  Resizes and pads an image to a target width and height.
+
+  Resizes an image to a target width and height by keeping
+  the aspect ratio the same without distortion and padding
+  it evenly with zeros.
```

Comment:
Mention in more detail what happens: use resize to resize to the closes (what's that) size that can be achieved without distortion, then add padding.